CHANGELOG.md

```diff
@@ -1,12 +1,12 @@
 7.1.0
 -----
+* Add new application_name configuration option
+* Update login page copy
```

Comment:
Can you remove these changes since they're not relevant to the PR. I'll just fix this on master.